index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute to Dr. Aklilu Lemma</title>
+  </head>
+  <body>
+    <main id="main">
+      <div id="heading" class="center">
+        <h1 id="title" class="center">Dr. Aklilu Lemma</h1>
+        <p class="font"><i>The man who was an Ethiopian pathobiologist</i></p>
+      </div>
+      <div id="img-div">
+        <img
+          class="center"
+          id="image"
+          src="Aklilu Lema.jpg"
+          alt="Aklilu Lemma with his Invention"
+        />
+        <div class="center">
+          <br />
+          <p id="img-caption" class="font">
+            Aklilu Lemma in his invention of Drugs...
+          </p>
+          <br />
+        </div>
+      </div>
+      <div class="div1">
+        <div id="tribute-info" class="center">
```

Comment:
It would be better if you use the section element before the main element because It combines portions of your document into discrete units that you can describe as related in some way.